src/components/Swap/AdvancedSwapDetails.tsx

```diff
@@ -70,6 +71,7 @@ export const TradeSummary: React.FC<TradeSummaryProps> = ({
     allowedSlippage,
   );
   const classes = useStyles();
+  const showRoute = Boolean(trade && trade.route.path.length > 2);
```

Comment:
Why not just always show the route?

Reply:
It shows the route when there is more than 2 items in the path in our current interface.